fastlane_bot/events/managers/manager.py

```diff
@@ -360,3 +360,21 @@ def handle_trading_fee_updated(self):
                 ]
                 pool["fee_float"] = pool["fee"] / 1e6
                 pool["descr"] = self.pool_descr_from_info(pool)
+
+    def update_remaining_pools(self):
+        remaining_pools = []
+        all_events = [pool[2] for pool in self.pools_to_add_from_contracts]
+        for event in all_events:
+            addr = self.web3.to_checksum_address(event["address"])
+            ex_name = self.exchange_name_from_event(event)
+            if not ex_name:
+                self.cfg.logger.warning("[run_async_update_with_retries] ex_name not found from event")
+                continue
+
+            key, key_value = self.get_key_and_value(event, addr, ex_name)
+            pool_info = self.get_pool_info(key, key_value, ex_name)
+
+            if not pool_info:
+                remaining_pools.append((addr, ex_name, event, key, key_value))
+
+        return remaining_pools
```

Comment:
No need to return `remaining_pools`, just use it for updating `self.pools_to_add_from_contracts` in this function

Reply:
fixed